### ISSUE TITLE
Two qubit RB:  target gate randomness

### ISSUE THREAD

**Issue by jaseung** [BBN-Q/QGL#166]
This is a weird issue and looks important to figure out as this could ruin two-qubit RB. Please see if this issue is reproducible to you.

I noticed that if I keep running a cell script in jupyter notebook to create two-qubit RB sequences with the same Clifford gate selection, I get some of target gates changing in the irregular way (See the picture attached below). So far I got four different gate combinations. I haven't checked if there are more of this randomness in other location yet.

To see this happen, try running the 4th cell in the jupyter notebook I attached multiple time. Alternatively, try running 5th and 6th cell in a row multiple times to see how actual pulse representation comes out.

Referring to [#165](https://github.com/BBN-Q/QGL/issues/162), I tried removing @_memoize from def CNOT, but I didn't see the issue fixed.

I am using the master branch of QGL.

Attached Files: 
1. Screenshot showing different output with the same Clifford gates.

![twoqubitrb_targetgaterandomness](https://user-images.githubusercontent.com/22563147/40553261-0fb935d4-6010-11e8-9463-8dfce9d23aef.PNG)

2. [TwoQubitRB_TargetGateRandomness.zip](https://github.com/BBN-Q/QGL/files/2039918/TwoQubitRB_TargetGateRandomness.zip)
This zip file has the jupyter notebook file that I used to observe this issue, and yml files.

**Comment by dieris** [comment]
I don't think this is an issue, these are different representations of the same Clifford, chosen randomly. See 
`Cliffords.C1Seqs` and https://github.com/BBN-Q/QGL/blob/ebcb7fa4d5475791873cf9073491cd10abe781de/QGL/Cliffords.py#L213
